src/Examine/LuceneEngine/LuceneSearchResults.cs

```diff
@@ -2,28 +2,16 @@
 using System.Collections;
 using System.Collections.Generic;
 using System.Linq;
-using System.Security;
-using Lucene.Net.Documents;
-using Lucene.Net.Search;
-using Examine.LuceneEngine.Providers;
 using Lucene.Net.Index;
+using Lucene.Net.Search;
 
 namespace Examine.LuceneEngine
 {
     /// <summary>
     /// An implementation of the search results returned from Lucene.Net
     /// </summary>
-    public class LuceneSearchResults : ISearchResults
+    public class LuceneSearchResults : LuceneSearchResultsBase
     {
-        ///<summary>
-        /// Returns an empty search result
-        ///</summary>
-        ///<returns></returns>
-        public static ISearchResults Empty()
```

Comment:
I think removing this will be a binary breaking change

Reply:
Indeed, you're probably right as it's now in a different namespace...

I guess best action is to keep this here but mark it obsolete?